quickwit/quickwit-search/src/metrics.rs

```diff
@@ -40,6 +42,7 @@ impl Default for SearchMetrics {
                 "Number of seconds required to run a leaf search over a single split. The timer \
                  starts after the semaphore is obtained.",
                 "search",
+                exponential_buckets(0.002, 2.0, 8).unwrap(),
```

Comment:
last bucket ends at half a second. while i understand smaller steps for leaf_search_split, in my experience having some split being longer than that isn't necessarily rare. imo either the start bucket or the number of bucket needs to be increased

Reply:
did both